app/models/concerns/arel_helper.rb

```diff
@@ -640,6 +644,10 @@ def as_t
       GrdaWarehouse::Hud::Assessment.arel_table
     end
 
+    def cas_t
+      Hmis::Hud::CustomAssessment.arel_table
```

Comment:
These should probably be moved to a hmis_arel_helper. There were already some Hmis records in here

Reply:
https://github.com/greenriver/hmis-warehouse/pull/2852/commits/17061083afbb3a3511c4e8cfe64e3213cf795d56